index.js

```diff
@@ -8,14 +8,30 @@ const notes = []
 app.use(parser)
 
 app.post('/notes', (req, res) => {
-  notes.push(req.body)
+  let idValue = notes.length
```

Comment:
Don't base your `id` off of the array length. Use a separate `nextId` variable and increment it after assigning it to a new `note`.